src/templates/base.html.twig

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>{% block title %}Week under control!{% endblock %}</title>
+        {% block stylesheets %}
+            <link rel="preconnect" href="https://fonts.googleapis.com">
+            <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+            <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@300;400;700&display=swap" rel="stylesheet">
+            <link href="{{ asset('fontawesome/css/fontawesome.css') }}" rel="stylesheet" />
+            <link href="{{ asset('fontawesome/css/brands.css') }}" rel="stylesheet" />
+            <link href="{{ asset('fontawesome/css/solid.css') }}" rel="stylesheet" />
```

Comment:
There are minified versions committed in your repository (e.g. `fontawesome.min.css`) - you should prefer those unless you have minification as part of your build process (they're smaller = faster load)